cartridges/openshift-origin-cartridge-python/metadata/manifest.yml

```diff
@@ -2,8 +2,8 @@
 Name: python
 Cartridge-Short-Name: PYTHON
 Display-Name: Python 2.6
-Description: ! 'Python is a general-purpose, high-level programming language whose
-  design philosophy emphasizes code readability. Popular development frameworks include:
+Description: 'Python is a general-purpose, high-level programming language whose design
+  philosophy emphasizes code readability. Popular development frameworks include:
```

Comment:
i'd go ahead and remove the apostrophes, none of the other carts have them anyway.